addons/website_sale/views/templates.xml

```diff
@@ -132,7 +132,7 @@
     </template>
 
     <template id="products_item" name="Products item">
-        <t t-set="product_href" t-value="request.httprequest.url_root + keep('shop/product/%s' % slug(product), page=(pager['page']['num'] if pager['page']['num']&gt;1 else None))" />
```

Comment:
Why were we using `request.httprequest.url_root` before ?

Reply:
It was used to ensure that google will reference the product with an absolute url. This is not necessary as google handle the absolute urls for structured datas. Tested it with this: https://search.google.com/structured-data/testing-tool/ and the url of the shop on the runbot.

Reply:
Indeed this does not seems to have any impact either it is a relative or absolute URL.
Tested also, including with translations url (fr_BE eg) and all good.
Btw, introduced for [this ticket](https://www.odoo.com/web#id=1774350&view_type=form&model=project.task&menu_id=) with [c7e9a8f74f2a82](https://github.com/odoo-dev/odoo/commit/c7e9a8f74f2a825eb4107b31fe283687331d17a6)